podman_compose.py

```diff
@@ -3239,9 +3239,11 @@ def compose_format_parse(parser):
         help="Pretty-print container statistics to JSON or using a Go template",
     )
 
-async def main():
+async def async_main():
     await podman_compose.run()
 
+def main():
+    asyncio.run(main())
```

Comment:
I guess this is a typo, you made it recursive

it should be

```
    asyncio.run(async_main())
```

but I don't think this is the problem

Reply:
yeap, that was a typo, to many main and async and main ;)

Thank you for the fix